src/onnx/onnx_parser.cpp

```diff
@@ -238,6 +257,26 @@ void onnx_parser::parse_from(const void* data, std::size_t size)
     }
 }
 
+void onnx_parser::parse_opset_version(const onnx::ModelProto& model)
```

Comment:
Can we make this a static function that returns the opset version: `std::size_t get_opset_version(const onnx::ModelProto& model)`? And then we can set the version with `opset_version = get_opset_version(model)`.

Reply:
changed accordly